client/state/data-layer/wpcom/theme-filters/index.js

```diff
@@ -34,16 +31,7 @@ const themeFiltersHandlers = dispatchRequest( {
 registerHandlers( 'state/data-layer/wpcom/theme-filters/index.js', {
 	[ THEME_FILTERS_REQUEST ]: [
 		( store, action ) => {
-			const state = store.getState();
-			const selectedSiteId = getSelectedSiteId( state );
-			const isFse = isSiteEligibleForLegacyFSE( state, selectedSiteId );
-			const isCoreFse = isSiteUsingCoreSiteEditor( state, selectedSiteId );
-
-			return themeFiltersHandlers( store, {
-				...action,
-				isFse,
-				isCoreFse,
-			} );
+			return themeFiltersHandlers( store, action );
```

Comment:
should we continue to add the `isCoreFse` info here?

Reply:
Nope, it's no longer being used at all. It was, for a time, used inside the `storeFilters` `onSuccess` callback to filter out, um, filters, that we didn't want to show (aka back when we didn't show `block-templates` to Classic sites).